src/components/ScreenWrapper/propTypes.js

```diff
@@ -44,6 +48,7 @@ const defaultProps = {
     onEntryTransitionEnd: () => {},
     modal: {},
     keyboardAvoidingViewBehavior: 'padding',
+    shouldEnableMaxHeight: !DeviceCapabilities.canUseTouchScreen(),
```

Comment:
Because the issue was reported mainly on web mobile, so I wanna leave the behavior for web as is just in case, but I think it can be `false`, should I change it?

Reply:
Updated